assets/src/components/d3/createHistogram.js

```diff
@@ -52,11 +52,19 @@ function createHistogram ({ data, width, height, domElement, xAxisLabel, yAxisLa
   // SVG Components
   const main = d3.select(domElement).append('div')
 
-  main.append('div')
+  const narrativeDiv = main.append('div')
     .attr('aria-live', 'polite')
     .attr('id', 'grade-view-narrative')
     .attr('class', 'fa-sr-only')
-    .text(d => narrativeTextGrades)
+    .text(d => 'The following paragraphs provide a text description for the graphical bar-chart on this page:')
+  narrativeDiv.append('span').text(d => narrativeTextGrades.courseStats)
+  narrativeDiv.append('span').text(d => narrativeTextGrades.binnedGradeText)
+  narrativeDiv.append('span').attr('aria-labelledby', 'grades-bins').text(d => 'Below is number of students in each grade range')
```

Comment:
```suggestion
  narrativeDiv.append('span').attr('aria-labelledby', 'grades-bins').text(d => 'List describing the number of students in each grade range')
```

We may want to check the language here with Jennifer.